src/DGtal/geometry/doc/moduleCurvatureMeasures.dox

```diff
@@ -0,0 +1,580 @@
+and Gaussian curvature measures) or tensors (anisotropic measures).
+
+A SurfaceMeshMeasure provides several handy methods to evaluate
+measures wrt simple subsets of \f$ \mathbb{R}^3 \f$.
+
+- SurfaceMeshMeasure::measure() const returns the total measure, i.e. \f$
+  \mu(\mathbb{R^3}) \f$.
+
+- SurfaceMeshMeasure::measure( const RealPoint&, Scalar, Face ) const
+  computes the measure \f$ \mu(B_r(x)) \f$, where \f$
+  B_r(x) \f$ is the ball of center \a x and radius \a r. The center \a
+  x must lie on or close to the face \a f.
+
+@note The measures may be associated to 0-, 1-, 2- cells, but
+generally you do not require this knowledge. If you prefer to have
+direct access to these measures, you have several overloaded methods
+to do this: SurfaceMeshMeasure::vertexMeasure,
+SurfaceMeshMeasure::edgeMeasure, SurfaceMeshMeasure::faceMeasure. They
+can accept weighted sum of cells as input.
+
+@subsection dgtal_curvature_measures_sec25 ICNC Curvature computation on OBJ surface
+
+Examples <a href="geometry_2meshes_2obj-curvature-measures-icnc-3d_8cpp-example.html">geometry/meshes/obj-curvature-measures-icnc-3d.cpp</a> and
+<a href="geometry_2meshes_2obj-curvature-measures-icnc-XY-3d_8cpp-example.html">geometry/meshes/obj-curvature-measures-icnc-XY-3d.cpp</a> show how to compute all curvature information on an arbitrary mesh.
+
+The only difference with examples above is the way the SurfaceMesh object is created. Here we read an OBJ file to create it.
+
+\snippet geometry/meshes/obj-curvature-measures-icnc-3d.cpp curvature-measures-SurfaceMesh
+
+Remember that a SurfaceMesh object may represent a non-manifold polygonal surface. 
+
+Once the mesh is created, you just need to ensure that the SurfaceMesh
+object has a normal vector attached to each of its vertex. This can be
+done as follows (or just provide an OBJ file with vertex normals):
+
+\snippet geometry/meshes/obj-curvature-measures-icnc-3d.cpp curvature-measures-CNC
+
+The rest of the code is identical. The two examples gives the
+following curvature estimates on "spot.obj" file.
+
+<table>
+<tr><td>
+\image html spot-cnc-H-r0_05.jpg "Interpolated corrected mean curvature measure, r=0.05" width=90%
+</td><td>
+\image html spot-cnc-G-r0_05.jpg "Interpolated corrected Gaussian curvature measure, r=0.05" width=90%
+</td><td>
+\image html spot-cnc-K1-D1-r0_05.jpg "Interpolated corrected smallest principal curvature and direction, r=0.05" width=90%
+</td><td>
+\image html spot-cnc-K2-D2-r0_05.jpg "Interpolated corrected greatest principal curvature and direction, r=0.05" width=90%
+</td></tr>
+</table>
+
+
+@section dgtal_curvature_measures_sec3 Corrected curvature measures on digital surfaces
+
+It is difficult to estimate curvatures on digital surfaces, since they
+naturally have only very few normal directions. You may have a look at
+module \ref moduleIntegralInvariant to see a method that estimates
+such curvatures when the digital surface is the boundary of a volume
+of voxels.
+
```

Comment:
It would be nice to provide a numerical (or timing) comparison between II and CNC for instance. Even just on an example (eg. Goursat, same ball radius and just a l_inf + timing error  measure). Could be a bit tricky to setup such test but it would be nice. 

(or to answer such questions: "If 10^-5 error is fine in the mean curvature estimate on a shape what is the fastest one?", or "if I have a time  budget if 10sec, which one is the more accurate?")

Reply:
(we may already have the data (+fig if needed) in the supplementary of the SGP paper).

Reply:
I have a look to see if I can code such a test case easily.

Reply:
Done in last commit.